src/scss/variables.scss

```diff
@@ -33,7 +33,7 @@ $zindex-drawing-toolbox: 3;
 $header-height: 3rem;
 $dev-disclaimer-height: 1.5rem;
 $footer-height: 1.5rem;
-$overlay-width: 360px;
+$overlay-width: 320px;
```

Comment:
@ltshb it seems that currently in cypress there is always a horizontal scroll bar for drawing features for $overlay-width = viewportWidth in cypress (only for chrome and electron not in firefox)